src/translation/translation_zh_CN.ts

```diff
@@ -119,7 +119,7 @@
     <message>
         <location filename="../util.cpp" line="745"/>
         <source>Norwegian Bokmål</source>
-        <translation>书面挪威语</translation>
+        <translation>노르웨이 보크몰어</translation>
```

Comment:
```suggestion
        <translation>书面挪威语</translation>
```

I guess this is a translation update that was submitted to the wrong language? The change is Korean but this change was submitted to Chinese language...

Reply:
Can you fix it on Weblate please?

Reply:
Done from the Weblate side.

Reply:
Thanks.